Project does not compile under linux

**Describe the bug**
Project does not compile under linux

**To Reproduce**
First, global.json pin sdk version to 2.1.403, which is outdated and makes it impossible(or difficult) to compile project out of the box. After removing `global.json` `dotnet build` starts but paket fails:
```
$ dotnet build Falanx.sln 
Microsoft (R) Build Engine version 15.9.20+g88f5fadfbe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  shasum: /home/vadim/projects/falanx/.paket/../paket-files/paket.restore.cached: 
  shasum: /home/vadim/projects/falanx/.paket/../paket-files/paket.restore.cached: 
  shasum: /home/vadim/projects/falanx/.paket/../paket-files/paket.restore.cached: 
  shasum: /home/vadim/projects/falanx/.paket/../paket-files/paket.restore.cached: 
  Paket version 5.189.1
  Starting full restore process.
  Stacktrace:
  
    at <unknown> <0xffffffff>
  
  Unhandled Exception:
  System.NullReferenceException: Object reference not set to an instance of an object
  [ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object
/home/vadim/projects/falanx/.paket/Paket.Restore.targets(91,5): error MSB3073: The command "mono --runtime=v4.0.30319 "/home/vadim/projects/falanx/.paket/paket.exe" restore" exited with code 1. [/home/vadim/projects/falanx/Falanx.Proto.Generator/Falanx.Proto.Generator.fsproj]
  Paket version 5.189.1
  Paket version 5.189.1
  Stacktrace:
  Stacktrace:
  
  
    at <unknown> <0xffffffff>
    at <unknown> <0xffffffff>
  
  
  Unhandled Exception:
  Unhandled Exception:
  System.NullReferenceException: Object reference not set to an instance of an object
  System.NullReferenceException: Object reference not set to an instance of an object
  [ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object
  [ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object
/home/vadim/projects/falanx/.paket/Paket.Restore.targets(91,5): error MSB3073: The command "mono --runtime=v4.0.30319 "/home/vadim/projects/falanx/.paket/paket.exe" restore" exited with code 1. [/home/vadim/projects/falanx/Falanx.Proto.Codec.Binary/Falanx.Proto.Codec.Binary.fsproj]
/home/vadim/projects/falanx/.paket/Paket.Restore.targets(91,5): error MSB3073: The command "mono --runtime=v4.0.30319 "/home/vadim/projects/falanx/.paket/paket.exe" restore" exited with code 1. [/home/vadim/projects/falanx/Falanx.Proto.Codec.Json/Falanx.Proto.Codec.Json.fsproj]
  Paket version 5.189.1
```

**Environement (please complete the following information):**
 - OS: linux:
```
$ lsb_release -irc
Distributor ID:	LinuxMint
Release:	18.3
Codename:	sylvia
```
 - Framework [e.g. netstandard2.0, net461]
```
$ dotnet --list-sdks
2.0.2 [/usr/share/dotnet/sdk]
2.1.500 [/usr/share/dotnet/sdk]
```

as a note, it is a paket error 

```
/home/vadim/projects/falanx/.paket/Paket.Restore.targets(91,5): error MSB3073: The command "mono --runtime=v4.0.30319 "/home/vadim/projects/falanx/.paket/paket.exe" restore" exited with code 1. [/home/vadim/projects/falanx/Falanx.Proto.Generator/Falanx.Proto.Generator.fsproj]
```

rerunning same command should give same error.

@vchekan the mono version?

--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -14,10 +14,6 @@
     <NupkgsDir>$([System.IO.Path]::GetFullPath("$(MSBuildThisFileDirectory)/artifact/nupkg"))</NupkgsDir>
     <MonoOrEmpty Condition=" '$(OS)' != 'Windows_NT' ">mono </MonoOrEmpty>
 
-    <!-- disable sourcelink on mono, to workaround https://github.com/dotnet/sourcelink/issues/155 -->
-    <EnableSourceLink Condition=" '$(OS)' != 'Windows_NT' AND '$(MSBuildRuntimeType)' != 'Core' ">false</EnableSourceLink>
-    <EnableSourceControlManagerQueries>$(EnableSourceLink)</EnableSourceControlManagerQueries>
-
     <Version Condition=" '$(Version)' == '' ">0.4.0-alpha3$(VersionSuffix)</Version>
 
     <NoWarn>$(NoWarn);FS2003</NoWarn>

--- a/paket.dependencies
+++ b/paket.dependencies
@@ -13,7 +13,7 @@ nuget Froto.Serialization
 nuget Fleece.NewtonsoftJson 0.8.0-alpha-2
 nuget Newtonsoft.Json 11.0.2
 
-nuget Microsoft.SourceLink.GitHub copy_local: true
+nuget Microsoft.SourceLink.GitHub 1.0.0-beta2-18618-05 copy_local: true
 
 github 7sharp9/FSharp.TypeProviders.SDK:generator src/ProvidedTypes.fs 
 github 7sharp9/FSharp.TypeProviders.SDK:generator src/ProvidedTypesTesting.fs 

--- a/paket.lock
+++ b/paket.lock
@@ -35,13 +35,13 @@ NUGET
     FSharp.Core (4.5.4)
     FSharpPlus (1.1.0-CI00252)
       FSharp.Core (>= 4.3.4)
-    Microsoft.Build.Tasks.Git (1.0.0-beta-63127-02) - copy_local: true
+    Microsoft.Build.Tasks.Git (1.0.0-beta2-18618-05) - copy_local: true
     Microsoft.NETCore.Platforms (2.1.1)
     Microsoft.NETCore.Targets (2.1)
-    Microsoft.SourceLink.Common (1.0.0-beta-63127-02) - copy_local: true
-    Microsoft.SourceLink.GitHub (1.0.0-beta-63127-02) - copy_local: true
-      Microsoft.Build.Tasks.Git (>= 1.0.0-beta-63127-02)
-      Microsoft.SourceLink.Common (>= 1.0.0-beta-63127-02)
+    Microsoft.SourceLink.Common (1.0.0-beta2-18618-05) - copy_local: true
+    Microsoft.SourceLink.GitHub (1.0.0-beta2-18618-05) - copy_local: true
+      Microsoft.Build.Tasks.Git (>= 1.0.0-beta2-18618-05)
+      Microsoft.SourceLink.Common (>= 1.0.0-beta2-18618-05)
     Microsoft.Win32.Primitives (4.3)
       Microsoft.NETCore.Platforms (>= 1.1)
       Microsoft.NETCore.Targets (>= 1.1)

